libc-test/build.rs

```diff
@@ -2266,6 +2266,8 @@ fn test_linux(target: &str) {
             // Require Linux kernel 5.x:
             | "MSG_COPY"
                if musl  => true,
+            // Require Linux kernel 5.1:
+            "F_SEAL_FUTURE_WRITE" => true,
```

Comment:
Sorry, I have a bit of a hard time understanding the match logic here (I'm not a very proficient in rust, yet). Should it be:

`"F_SEAL_FUTURE_WRITE" => true,`

or

`| "F_SEAL_FUTURE_WRITE" => true,`

?

Reply:
The first `|` is optional: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=3007c8d7c8a3b7d425ce1580c8cc51ed

Reply:
I see, thanks for the explanation.